site/gdocs/components/centered-article.scss

```diff
@@ -615,8 +625,26 @@ h3.article-block__heading.has-supertitle {
     .centered-article-header__byline,
     .centered-article-header__dateline {
         // A small bit of padding to encourage line-wrapping on long bylines
```

Comment:
The padding is no longer there. Could you update the comment, if it's still valid?